wstring.hh

```diff
@@ -55,8 +55,8 @@ namespace gd
 
   #else
 
-  typedef wchar_t wchar;
-  using std::wstring;
+  typedef char32_t wchar;
+  typedef std::u32string wstring;
```

Comment:
Changes to wstring.hh is not releated to the TOPIC of this pull request!！
Please make sure you are realy ready to make the request easily acceptable just as you do in #1447.

Reply:
iconv changed to qstream .   u32string was an easiler choice.
QString::toStdU32String()

Reply:
>u32string was an easiler choice. QString::toStdU32String()

Yes, it's off this TOPIC.